sys-auth/elogind/elogind-255.5-r1.ebuild

```diff
@@ -172,6 +172,18 @@ pkg_postinst() {
 			elog "configuration remember to migrate those to new configuration file."
 		fi
 	done
+
+	local file files
+	# find custom hooks excluding known (nvidia-drivers, sys-power/tlp)
+	readarray -t files <<<$(find /$(get_libdir)/elogind/system-sleep/ -type f \
+		\( -not -iname ".keep_dir" -a -not -iname "nvidia" -a -not -iname "49-tlp-sleep" \))
```

Comment:
fwiw may want either a 2>/dev/null or check if the directory exists first or else get a noisy:
`find: ‘/lib64/elogind/system-sleep/’: No such file or directory`